status-im/status-desktop
# fix: Show only selected networks

Closes #16534

### What does the PR do

* Removed unused component: `NetworkSelectionView`
* Updated Assets details header to show only enabled networks
* id rename changes

### Affected areas

Assets view

### Architecture compliance

My PR is consistent with this document: [Status Desktop Architecture Guide](https://github.com/status-im/status-desktop/blob/master/CONTRIBUTING.md)

### Screenshot of functionality (including design for comparison)


https://github.com/user-attachments/assets/430d29d0-f209-44af-a98a-d755c54aba8f


### Impact on end user

What is the impact of these changes on the end user (before/after behaviour)

### How to test

- How should one proceed with testing this PR.
Selected preffered  network, go to any asset details
- What kind of user flows should be checked?
Network selection in wallet view

### Risk 

Described potential risks and worst case scenarios.

Tick **one**:
